pages/addhabit/index.js

```diff
@@ -0,0 +1,14 @@
+import AddHabitForm from "@/components /AddHabitsForm";
```

Comment:
your components folder is called `components ` (yes, ending with a space) please rename your folder to not end with a space and then do a search and replace to change all instances of

`@/components /` with `@/components/`

```suggestion
import AddHabitForm from "@/components/AddHabitsForm";
```